docs/docs-content/clusters/public-cloud/aws/eks-hybrid-nodes/create-hybrid-node-pools.md

```diff
@@ -0,0 +1,370 @@
+
+### Validate
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+2. From the left **Main Menu**, select **Clusters**.
+
+3. Select your cluster to view its **Overview** tab.
+
+4. Select the **Nodes** tab.
+
+5. Your newly added hybrid node pool displays as **Running**.
+
+   ![An active hybrid node pool on the Nodes tab](/aws_eks-hybrid_create-hybrid-node-pools_running-hybrid-pool.webp)
+
+## Configure Hybrid Node Networking for VPN Solutions
+
+This section explains how to configure your VPN with static routes to enable network traffic to reach pods on hybrid
+nodes. Before proceeding, consider the following points:
+
+- This guide is specifically for VPN solutions that support and require manual static route configuration. If your VPN
+  uses a different routing mechanism, these steps may not apply.
+
+- If your VPN supports BGP (Border Gateway Protocol), you may be able to skip manual route configuration entirely. BGP
+  can automatically advertise and update routes between your cluster and VPN. Check your VPN's documentation for BGP
+  neighbor configuration and route import procedures.
+
+- These steps outline the general process for configuring a VPN with static routes. Refer to your VPN's documentation
+  for provider-specific configuration details.
+
+### Prerequisites
+
+- Your hybrid node pool has been created and configured. Refer to [Create Hybrid Node Pool](#create-hybrid-node-pool)
+  for guidance.
+
+- Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/) is installed and available in your local workstation.
+
+- Access to your Amazon EKS cluster through kubectl.
+
+  - To access your cluster with kubectl, you can use the AWS CLI's built-in authentication capabilities. If you are
+    using a custom OIDC provider, you will need to configure your kubeconfig to use your OIDC provider.
+
+    Refer to the
+    [Access Imported Cluster with Kubectl](./import-eks-cluster-enable-hybrid-mode.md#access-imported-cluster-with-kubectl)
+    section for more information.
+
+- Access to your VPN configuration interface.
+
+### Configure Networking
+
+1. Issue the following kubectl command to list all CiliumNode resources in your cluster.
+
+   ```shell
+   kubectl get ciliumnode --output wide
+   ```
+
+   Example output.
+
+   ```shell
+   NAME                                    CILIUMINTERNALIP   INTERNALIP    AGE
+   edge-abc123def4567890example1           192.168.5.101      10.200.1.23   2h
+   edge-xyz987uvw6543210example2           192.168.6.102      10.200.2.34   3h
+   ```
+
+2. For each hybrid node, retrieve the `spec.ipam.podCIDRs` field to find the CIDR block allocated for pods active on
+   that node.
+
+   Replace `<nodeName>` with the name of your hybrid node discovered in step 1. Repeat this step for each hybrid node
+   found.
+
+   ```shell
+   kubectl get ciliumnode <nodeName> --output yaml | grep "podCIDRs" --after-context 1
+   ```
+
+   Example output.
+
+   ```shell
+       podCIDRs:
```

Comment:
I think this block should be unindented.

Reply:
It's indented in the terminal output, so I left that indentation in place.